src/Kunstmaan/GeneratorBundle/Command/InstallCommand.php

```diff
@@ -0,0 +1,156 @@
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Output\OutputInterface;
+use Sensio\Bundle\GeneratorBundle\Command\Validators;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Process\Exception\RuntimeException;
+use Sensio\Bundle\GeneratorBundle\Command\Helper\QuestionHelper;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+
+/**
+ * Kunstmaan installer
+ */
+class InstallCommand extends Command
+{
+    /**
+     * @var int
+     */
+    private $commandSteps = 0;
+
+    /** @var string */
+    private $rootDir;
+
+    /**
+     * @param string $rootDir
+     */
+    public function __construct(string $rootDir)
+    {
+        $this->rootDir = $rootDir;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('kuma:install')
+            ->setDescription('KunstmaanCMS installer.')
+            ->setDefinition(
+                new InputDefinition(
+                    [
+                        new InputOption('namespace', '', InputOption::VALUE_REQUIRED, 'The namespace of the bundle to create'),
+                        new InputOption('demosite', '', InputOption::VALUE_REQUIRED, 'Do you want create "demosite"'),
+                        new InputOption('dir', '', InputOption::VALUE_REQUIRED, 'The directory where to create the bundle'),
+                        new InputOption('bundle-name', '', InputOption::VALUE_REQUIRED, 'The optional bundle name'),
+                    ]
+                )
+            );
+    }
+
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        $questionHelper = new QuestionHelper();
+
+        $outputStyle = new SymfonyStyle($input, $output);
+        $outputStyle->writeln('<info>Installing KunstmaanCms...</info>');
+        $outputStyle->writeln($this->getKunstmaanLogo());
+
+        $question = new Question(
+            $questionHelper->getQuestion('Bundle namespace', $input->getOption('namespace')),
+            $input->getOption('namespace')
+        );
+        $question->setValidator([Validators::class, 'validateBundleNamespace']);
+        $namespace = $questionHelper->ask($input, $output, $question);
+        $input->setOption('namespace', $namespace);
+
+        $question = new ChoiceQuestion(
+            'Do you want create "demosite"',
+            ['No', 'Yes'],
+            0
+        );
+        $question->setErrorMessage('Option "%s" is invalid.');
+        $demositeOption = $questionHelper->ask($input, $output, $question);
+
+        $input->setOption('demosite', $demositeOption);
+        $input->setOption('bundle-name', strtr($namespace, ['\\Bundle\\' => '', '\\' => '']));
+
+        $dir = $input->getOption('dir') ?: dirname($this->rootDir) . '/src';
```

Comment:
When the service parameter is changed from `root_dir` to `project_dir`, the dirname function should be removed